db/migrate/20220222135646_add_index_to_tables.rb

```diff
@@ -0,0 +1,11 @@
+class AddIndexToTables < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
```

Comment:
really good job)
tell your colleagues about disable_ddl_transaction! and algorithm: :concurrently
on a real project it is unlikely that there will be such a migration, but if suddenly, it is better to have add indexes in separate migration files.

Reply:
thanks, the strong_migrations gem and your articles helped
I decided not to clog the migration, but I will know